src/plugin.js

```diff
@@ -70,7 +72,7 @@ module.exports = class IconPlugin {
               // I would've just used compiled.options.output.publicPath here,
               // but apparently <use> tags don't yet support cross-origin
               // requests. This should be made configurable.
-              `/static/${this.getFilename()}`
+              `${publicPath}${this.getFilename()}`
```

Comment:
So the problem here, and the reason I didn't do it this way in the first place, is that `publicPath` often points to a CDN host, and `<use>` tags don't support cross-origin requests. I was thinking that this should be a separate configuration option passed to the class's constructor.